runtime-sdk/modules/evm/src/precompile/utils.rs

```diff
@@ -0,0 +1,62 @@
+//! Utilities useful in implementing precompiles.
+use std::cmp::min;
+
+use evm::{executor::stack::PrecompileFailure, ExitError};
+use num::{BigUint, ToPrimitive as _};
+
+/// Length of an EVM word, in bytes.
+pub const WORD: usize = 32;
+
+pub fn round_up(what: usize) -> usize {
```

Comment:
```suggestion
pub fn round_up_nyble(what: usize) -> usize {
```

Reply:
Depends on what you mean by the u5. The function should round the number up to the nearest multiple of 32, so the bottom 5 bits are cleared.